book/preliminary/git.md

```diff
@@ -60,15 +61,17 @@ credential.helper=store
 ```
 
 ### 4. Authenticating with GitHub
-In order to interact with GitHub via the `git' command inside the JupyterHub environment, 
-you need to set up an authentication method. One way to do this is to use your GitHub password. However, it is a better, more secure practice to use a GitHub personal access token.
-+++
+In order to interact with GitHub via the `git` command inside the JupyterHub environment,
+you need to set up an authentication method. One way to do this is to use your GitHub password.
+However, it is a better, more secure practice to use a GitHub personal access token because they
+have more limited permissions and can be easily revoked.
```

Comment:
This is a nit-picky comment (with my IT security hat on) and I would update the wording to:
```
more customizable permissions and allow to revoke account access without affecting your GitHub user account password
```

You can still do a lot of damage with tokens and I would not want to create a false sense of security with the use of tokens.

Reply:
good point! I modified that line